code/modules/surgery/tools.dm

```diff
@@ -292,7 +293,8 @@
 		icon_state = "retractor_a"
 
 /obj/item/retractor/advanced/examine()
-	. += " It resembles a retractor[tool_behaviour == TOOL_RETRACTOR ? "retractor" : "hemostat"]."
+	. = ..()
+	. += " It resembles a [tool_behaviour == TOOL_RETRACTOR ? "retractor" : "hemostat"]."

```

Comment:
```suggestion
	. += " It resembles a [tool_behaviour]."
```

Reply:
Alternatively don't do that.
This is fine as it is. I would prefer is the actual value behind the tool defines didn't matter.

Should a maintainer disagree with me on this, at the VERY least leave a comment about this in the _DEFINES file.